<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsWidget</class>
 <widget class="QWidget" name="GeneralSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>Choose which version of Epub to use
when creating new or empty Epubs in Sigil.</string>
     </property>
     <property name="title">
      <string>Create New or Empty Epubs as:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout2">
      <item>
       <widget class="QRadioButton" name="EpubVersion2">
        <property name="toolTip">
         <string>Epub Version 2.</string>
        </property>
        <property name="text">
         <string>Version 2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="EpubVersion3">
        <property name="toolTip">
         <string>Epub Version 3.</string>
        </property>
        <property name="text">
         <string>Version 3</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="toolTip">
      <string>Choose when your HTML code is automatically 
mended.</string>
     </property>
     <property name="title">
      <string>Mend XHTML Source Code On:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="MendOnOpen">
        <property name="toolTip">
         <string>Mend when opening an Epub or HTML file, 
and when switching from Book View to Code View.</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="MendOnSave">
        <property name="toolTip">
         <string>Mend when saving an Epub.</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="toolTip">
      <string>Determine if Epubs are allowed to access non-multimedia remote resources.</string>
     </property>
     <property name="title">
      <string> Control Access by Epubs to non-multimedia remote resources.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="AllowRemote">
        <property name="toolTip">
         <string>Check to allow Epubs to access non-multimedia remote resources.</string>
        </property>
        <property name="text">
         <string>Epubs may access all remote resources types.</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>What CSS level to specify for W3C stylesheet validation tool (EPUB2)</string>
     </property>
     <property name="title">
      <string>EPUB2 W3C Stylesheet Validation Level:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutcss2">
      <item>
       <widget class="QRadioButton" name="Epub2css20">
        <property name="toolTip">
         <string>W3C stylesheet validator will use CSS level 2 for EPUB2</string>
        </property>
        <property name="text">
         <string>CSS Level 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Epub2css21">
        <property name="toolTip">
         <string>W3C stylesheet validator will use CSS level 2.1 for EPUB2</string>
        </property>
        <property name="text">
         <string>CSS Level 2.1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Epub2css30">
        <property name="toolTip">
         <string>W3C stylesheet validator will use CSS level 3 for EPUB2</string>
        </property>
        <property name="text">
         <string>CSS Level 3</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>What CSS Level to specify for W3C stylesheet validation tool (EPUB3)</string>
     </property>
     <property name="title">
      <string>EPUB3 W3C Stylesheet Validation Level:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutcss3">
      <item>
       <widget class="QRadioButton" name="Epub3css20">
        <property name="toolTip">
         <string>W3C stylesheet validator will use CSS level 2 for EPUB3</string>
        </property>
        <property name="text">
         <string>CSS Level 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Epub3css21">
        <property name="toolTip">
         <string>W3C stylesheet validator will use CSS level 2.1 for EPUB3</string>
        </property>
        <property name="text">
         <string>CSS Level 2.1</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Epub3css30">
        <property name="toolTip">
         <string>W3C stylesheet validator will use CSS level 3 for EPUB3</string>
        </property>
        <property name="text">
         <string>CSS Level 3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Set Folder where temporary files should be created</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="autoButton">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
